src/engraving/rendering/dev/beamlayout.cpp

```diff
@@ -1014,7 +1016,11 @@ void BeamLayout::createBeamSegments(Beam* item, const LayoutContext& ctx, const
                                                                    previousBreak32);
                         createBeamletSegment(item, ctx, toChord(startCr), isBeamletBefore, level);
                     } else {
-                        createBeamSegment(item, startCr, endCr, level);
+                        createBeamSegment(item, startCr, endCr, level, shortStems, alteredCrs);
```

Comment:
It isn't very clear to me what these `alteredCrs` represent. Could there be a more descriptive name for this variable? In the worst case, a short comment could do